### PR TITLE
Bitcoin Core 0.20.1

### PR DESCRIPTION
**Pull request by laanwj**
Can be merged if it passes Travis.

### CODE REVIEW

**Comment by Cobra-Bitcoin** [comment]
Not sure why test is failing, will investigate. Good job on the release.

File: _releases/v0.20.1.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  stop-starting your node.
+
+Notification changes
+--------------------
+
+`-walletnotify` notifications are now sent for wallet transactions that are
+removed from the mempool because they conflict with a new block. These
+notifications were sent previously before the v0.19 release, but had been
+broken since that release (bug
+[#18325](https://github.com/bitcoin/bitcoin/issues/18325)).
```
Comment (Cobra-Bitcoin) [inline _releases/v0.20.1.md:127]: Maybe this extra bracket at the end is causing the test to break?

**Comment by jdabs** [comment]
I think the githubify tags are not getting translated.

**Comment by sipa** [comment]
@laanwj Ping?

File: _releases/v0.20.1.md (hunk 1 above)
Comment (laanwj) [reply _releases/v0.20.1.md:127]: Thanks, will remove it and retry.

**Comment by laanwj** [comment]
Whew, passing now.

**Pull request merged**